core.go

```diff
@@ -229,7 +229,8 @@ func (sock *socket) SendMsg(msg *Message) error {
 }
 
 func (sock *socket) Send(b []byte) error {
-	msg := &Message{Body: b, Header: nil, refcnt: 1}
+	msg := NewMessage(len(b))
```

Comment:
Thinking about this, there is no value in setting the size len(b), because we're not using the body that NewMessage() allocates.  Instead, just pass 0.  This will probably reduce pressure on the Heap even further, and also benefits from accessing the larger cache of messages.  (Recall we limit how many Messages can be cache allocated from each cache, with fewer of them at the larger buffer sizes.)